File: src/main/java/com/example/himaps/core/AP.java
package com.example.himaps.core;

public class AP {
    public String bssid;
    public String ssid;
    public int strength;

    public AP(String bssid, String ssid, int strength) {
        this.bssid = bssid;
        this.ssid = ssid;
        this.strength = strength;
    }
}

File: wwwroot/ap.php
<?php


class ap {
    public String $bssid;
    public String $ssid;
    public int $strength;

    /**
     * ap constructor.
     * @param String $bssid
     * @param String $ssid
     * @param int $strength
     */
    public function __construct(string $bssid, string $ssid, int $strength)
    {
        $this->bssid = $bssid;
        $this->ssid = $ssid;
        $this->strength = $strength;
    }

}
